optaplanner-webexamples/src/main/resources/org/optaplanner/webexamples/vehiclerouting/belgium-road-time-n40-k10.vrp

```diff
@@ -0,0 +1,174 @@
+NAME: belgium-road-time-n50-k10
```

Comment:
Do we have to make changes to DepotDistanceCustomerDifficultyWeightFactory and DepotAngleCustomerDifficultyWeightFactory? 
The weights are calculated based on the first Depot.